hrms/payroll/doctype/payroll_entry/test_payroll_entry.py

```diff
@@ -749,9 +749,20 @@ def test_loan_repayment_from_salary(self):
 			total_loan_repayment=loan.monthly_repayment_amount,
 		)
 
+		salary_slip_name = frappe.db.get_value("Salary Slip", {"payroll_entry": payroll_entry.name}, "name")
+		salary_slip = frappe.get_doc("Salary Slip", salary_slip_name)
+		payroll_entry.reload()
+
+		initial_gross_pay = flt(salary_slip.gross_pay) - flt(salary_slip.total_deduction)
+		loan_repayment_amount = flt(salary_slip.total_loan_repayment)
+		expected_net_pay = initial_gross_pay - loan_repayment_amount
+
 		payroll_entry.make_bank_entry()
 		submit_bank_entry(payroll_entry.name)
 
+		salary_slip.reload()
+		self.assertEqual(salary_slip.net_pay, expected_net_pay)
```

Comment:
Thanks for updating the readability fix!

But in the test here — this was already working as expected, IG? The main issue was with bank entry's amount from payroll entry and not the net pay in salary slip if I understand correctly. Can we test that instead? 🙊